lib/fractal/govuk-theme/index.js

```diff
@@ -36,10 +36,10 @@ const customTheme = mandelbrot({
 })
 
 // Specify a template directory to override any view templates
-customTheme.addLoadPath('./fractal/govuk-theme/views/')
+customTheme.addLoadPath('lib/fractal/govuk-theme/views/')
```

Comment:
I've updated this to use `__dirname`, rather than paths, as long term we'd probably want the theme to be a separate package, so avoid dependency on paths.